installation-manager-core/src/main/java/com/codenvy/im/report/ReportType.java

```diff
@@ -32,8 +32,7 @@ public ReportParameters getParameters() {
         ReportParameters parameters = new ReportParameters(
```

Comment:
No, we can't, so as in this case the value of **parameterPrefix** is unknown at the build time.